packages/ketcher-react/src/script/ui/data/schema/struct-schema.js

```diff
@@ -155,6 +155,7 @@ export const bond = {
     type: {
       title: 'Type',
       enum: [
+        '',
```

Comment:
Looks strange

Reply:
@AnastasiiaPlyako can you please comment on this

Reply:
Discussed with Konstantin. This field allows you to have an empty line when selected bonds have different types. The same logic is realized for atoms. https://github.com/epam/ketcher/blob/master/packages/ketcher-react/src/script/ui/data/schema/struct-schema.js#L62